util/error.go

```diff
@@ -2,13 +2,18 @@ package util
 
 import (
 	"errors"
+	"fmt"
+	"time"
+
 	"github.com/aws/aws-sdk-go/aws/awserr"
 )
 
 var ErrInSufficientPermission = errors.New("error:INSUFFICIENT_PERMISSION")
 var ErrDifferentOwner = errors.New("error:DIFFERENT_OWNER")
+var ErrContextExecutionTimeout = errors.New("error:EXECUTION_TIMEOUT")
 
 const AWsUnauthorizedError string = "UnauthorizedOperation"
+const AwsDryrunSuccess string = "Request would have succeeded, but DryRun flag is set."
```

Comment:
I suspect it should be `AwsDryRunSuccess`, IDE is complaining about `AwsDryrunSuccess`